### PR TITLE
chore(deps): dependencies bumps + lint fixes

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Linting errors

## Resolution
* Various lint fixes from across all the micro-services
* Dependencies updates

### CODE REVIEW

File: e2e-tests/gef/cypress/fixtures/mocks/mock-user-maker.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 exports.MOCK_USER_MAKER = {
   username: 'BANK1_MAKER1',
   password: 'AbC!2345',
-  firstname: 'Tamil',
-  surname: 'Rahani',
+  firstname: 'First',
```
Comment (IgnatG) [inline e2e-tests/gef/cypress/fixtures/mocks/mock-user-maker.js:4]: it may not be part of this PR, but should we start using the `chance` package for these values? i.e. `chance.first()`

Comment (abhi-markan) [reply e2e-tests/gef/cypress/fixtures/mocks/mock-user-maker.js:4]: Good shout I'll raise a new PR once this is merged

**Review by IgnatG — APPROVED** [review]
LGTM

**Review by IgnatG — APPROVED** [review]
